lib/models/ui/create_account_errors.dart

```diff
@@ -0,0 +1,40 @@
+import "package:flutter/foundation.dart";
+
+/// A dataclass handling the error status of the create account page.
+@immutable
+class CreateAccountErrors {
+  final String? uniqueUsernameError;
+  final String? pseudoError;
+  final bool valid;
+
+  /// Create a new instance of [CreateAccountErrors].
+  /// [uniqueUsernameError] and [pseudoError] are the error messages for
+  /// the unique username and pseudo text fields respectively. If they are null,
+  /// no error is displayed. [valid] is true if the form is valid. It is here to
+  /// differentiate from the case where the fields have no error because they have
+  /// not yet been validated from the case they have no errors because they are valid.
+  const CreateAccountErrors({
+    required this.uniqueUsernameError,
+    required this.pseudoError,
+    required this.valid,
+  });
```

Comment:
I wonder if we could add some default values in the constructor so that we don't have to specify all the fields if we only want to show a uniqueUsernameError for instance.

With the previous suggestion in mind, we could have:
```suggestion
  const CreateAccountErrors({
    this.uniqueUsernameError,
    this.pseudoError,
    this.created = false,
  });
```

Reply:
This appeared to be useful in my factoring indeed. Fixed in 02307c481556be376b9a579679970839efef6ed6.